easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.19-GCC-6.4.0-2.28-LAPACK-3.7.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenBLAS'
+version = '0.2.19'
```

Comment:
OpenBLAS 0.2.20 is expected to be released soon (see https://github.com/xianyi/OpenBLAS/issues/1223); I suggest we postpone setting `foss/2017b` in stone until it's available?